pep/app/pods/components/page/sidebar/widgets/more-like-these/component.ts

```diff
@@ -1,18 +1,68 @@
 import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
 
-import AjaxService from 'pep/services/ajax';
 import { PageSidebarWidgetArgs } from 'pep/pods/components/page/sidebar/widgets/component';
 import { WIDGET } from 'pep/constants/sidebar';
+import Document from 'pep/pods/document/model';
+import { dontRunInFastboot } from 'pep/decorators/fastboot';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import SimilarityMatch from 'pep/pods/similarity-match/model';
+import { DS } from 'ember-data';
+import { restartableTask } from 'ember-concurrency-decorators';
+import { taskFor } from 'ember-concurrency-ts';
 
 interface PageSidebarWidgetsMoreLikeTheseArgs extends PageSidebarWidgetArgs {}
 
 export default class PageSidebarWidgetsMoreLikeThese extends Component<PageSidebarWidgetsMoreLikeTheseArgs> {
-    @service ajax!: AjaxService;
+    @service store!: DS.Store;
+    @tracked results?: SimilarityMatch;
+    similarCount = 5;
+
+    get data(): Document {
+        return this.args.data[this.widget];
+    }
 
     get isOpen() {
         return this.args.openWidgets.includes(this.widget);
     }
 
     widget = WIDGET.MORE_LIKE_THESE;
+
+    /**
+     * Load the similar from the document they are reading
+     */
+    @restartableTask
+    *loadSimilarFromDocument() {
+        if (this.data?.id) {
+            const results = yield this.store.findRecord('document', this.data.id, {
+                reload: true,
+                adapterOptions: { query: { similarcount: this.similarCount } }
+            });
+            const matches = results.similarityMatch.similarDocuments.filter(
+                (item: Document) => item.id !== this.data.id
+            );
+
+            // TODO Right now the document that the find was called for is somehow being added into
+            // the array. We should figure out why and fix it
```

Comment:
Yea there is a good chance it is, I just didn't have any more time to fool around with it. I was seeing the similar docs include the document that we are getting the similar docs for.

Reply:
hm yeah, it might be that you need to use some of the relationship configs, like `inverse: null` or something along those lines to tell ember-data how to treat the relationship. 

https://guides.emberjs.com/release/models/relationships/#toc_explicit-inverses

I noticed you had some pretty funky stuff going on in the serializer too, so thats another place i'd probably examine more closely too if i were to look at it.